src/main/java/org/sg/backing/TopicController.java

```diff
@@ -0,0 +1,87 @@
+package org.sg.backing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.faces.bean.ManagedBean;
+import javax.faces.bean.SessionScoped;
+
+import org.sg.domain.Topic;
+
+@ManagedBean
+@SessionScoped
+public class TopicController {
+	private List<Topic> topicList = new ArrayList<Topic>();
+	private Topic selectedTopic;
+	
+	private int newId;
+	private String newName;
+	private String newDescription;
+	private int newCourse;
```

Comment:
Un Topic e' associato a più di un Corso. Nel modulo Topic io gestirei i topics senza pensare in che Corso appratiranno. Poi dai Corsi, fare una lista di Topics.

Reply:
stesso discorso del id course